web/modeltr/connection.py

```diff
@@ -35,7 +38,7 @@ def __init__(self, **kwargs):
         for i in range(Settings.app['retries']['db_connection']):
             try:
                 self.async_mode = True if 'async_mode' in kwargs else False
```

Comment:
Could be simplified like this:
```suggestion
                self.async_mode = 'async_mode' in kwargs
```